client/src/styles/components/_chat.scss

```diff
@@ -264,4 +263,46 @@
       }
     }
   }
+
+  .parent {
+    position: fixed;
+    left: 0.8rem;
+    bottom: 10rem;
+
+    .scroll-button {
+      position: relative;
+    }
+
+    .circle {
+      color: #9360ff;
+      position: absolute;
+      top: 50%;
+      left: 50%;
+    }
+
+    .arrow {
+      color: white;
+      position: absolute;
+      top: 50%;
+      left: 50%;
+      transform: translate(25%, 25%);
+    }
+
+    .new-msgs {
+      background-color: #0159ff;
+      border-color: #0159ff;
+      color: white;
+      font-size: 0.75rem;
+      font-weight: bold;
+      position: absolute;
+      transform: translate(60%, -50%);
+      left: 50%;
+      -moz-border-radius: 20px;
+      border-radius: 20px;
+      padding-left: 8px;
+      padding-right: 8px;
+      padding-top: 5px;
+      padding-bottom: 5px;
```

Comment:
Not that critical but it makes it more clean: condense it into the padding shorthand property